core/src/main/scala/org/apache/spark/scheduler/SchedulableBuilder.scala

```diff
@@ -191,7 +191,10 @@ private[spark] class FairSchedulableBuilder(val rootPool: Pool, conf: SparkConf)
         parentPool = new Pool(poolName, DEFAULT_SCHEDULING_MODE,
           DEFAULT_MINIMUM_SHARE, DEFAULT_WEIGHT)
         rootPool.addSchedulable(parentPool)
-        logInfo("Created pool: %s, schedulingMode: %s, minShare: %d, weight: %d".format(
+        logWarning(("An Unconfigured pool is found. It is built with default configuration. " +
```

Comment:
It's not totally clear what the first part of this error message means.  How about changing the message to:

```
logWarning(s"A job was submitted with scheduler pool $poolName, which has not been configured." +
    "This can happen when the file that pools are read from isn't set, or when that file " +
    s"doesn't contain $poolName.  Created $poolName with default configuration (" +
    s"schedulingMode: $DEFAULT_SCHEDULING_MODE, minShare: $DEFAULT_MINIMUM_SHARE, " +
    s"weight: $DEFAULT_WEIGHT)")
```